pkg/server/definitionHandler.go

```diff
@@ -0,0 +1,218 @@
+package server
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"go.etcd.io/etcd/pkg/fileutil"
```

Comment:
do we really want to import the whole etcd libariry for a util function?

Reply:
No, I can use `os.stat` to check whether the directory exists. Thanks.